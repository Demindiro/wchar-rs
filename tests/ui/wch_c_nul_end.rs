use wchar::wch_c;

const NUL_END: &[u16] = wch_c!("nul byte end\0");

fn main() {}
