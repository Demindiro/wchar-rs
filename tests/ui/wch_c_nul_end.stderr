error: C-style string cannot contain nul characters
 --> $DIR/wch_c_nul_end.rs:3:32
  |
3 | const NUL_END: &[u16] = wch_c!("nul byte end\0");
  |                                ^^^^^^^^^^^^^^^^
